capsule_brain/tools/aggregator.py

```
@@ -1,33 +1,108 @@
+
+
+def _safe_eval(expr: str) -> float:
+    """Evaluate a simple arithmetic expression safely."""
+
+    node = ast.parse(expr, mode="eval").body
+
+    def eval_(ast_node: ast.AST) -> float:
+        if isinstance(ast_node, ast.Num):  # type: ignore[attr-defined]
+            literal = ast_node.n
+            if not isinstance(literal, (int, float)):
+                raise ValueError("Unsupported literal")
```

Ruff (0.12.2)

Prefer `TypeError` exception for invalid type

(TRY004)